st.js

```diff
@@ -219,7 +219,7 @@ class Mount {
     // trailing slash removal to fix Node.js v23 bug
     // https://github.com/nodejs/node/pull/55527
     // can be removed when this is resolved and released
-    return path.join(this.path, u.replace(/\/+$/, ''))
+    return path.join(this.path, u.replace(/(?<!\/)\/+$/, ''))
```

Comment:
I think the negative look-behind gets us in the wrong place here because it'll not match multiple slashes, voiding the `+` all together. I think the right answer here is to just avoid regexes entirely; plus the original comment should be changed since we've now essentially baked in normalised paths that help us get consistent rendering when returning a directory listing.


```suggestion
    // Normalise paths by removing trailing slashes
    // This ensures consistent paths for directory content rendering
    while (u.length > 0 && u[u.length - 1] === '/') {
      u = u.slice(0, -1)
    }
    return path.join(this.path, u)
```

Reply:
Ping @rvagg